orientation/exercises/10_TEST_ANIMALS.md

```diff
@@ -19,8 +19,9 @@ As a team, we'll be building unit test coverage for all the functionality of the
 
 Write test cases to verify the I/O of the following methods of `Animal` and `Dog`.
 
+<!-- TODO: IN my code, I never create an instance of Animal. Is it necessary? -->
```

Comment:
Yes, because there are requirements below to write tests for the `Animal` instance.

Reply:
👍